src/hotspot/os/linux/globals_linux.hpp

```diff
@@ -114,4 +114,9 @@ define_pd_global(bool, UseLargePages, false);
 define_pd_global(bool, UseLargePagesIndividualAllocation, false);
 define_pd_global(bool, UseThreadPriorities, true) ;
 
+// On some systems using SSSD files in this directory are left open
+// after calling getpwuid_r, getpwname_r, getgrgid_r, getgrname_r
+// or other functions in this family.
+define_pd_global(ccstrlist, CRAllowedOpenFilePrefixes, "/var/lib/sss/mc/");
```

Comment:
This have to be replicated over all platforms, e.g. macos build is broken https://github.com/rvansa/crac/actions/runs/6784349063/job/18440498579

Reply:
Spot on. For some reason my VSCode left those files unsaved (and therefore not comitted).